meson.build

```diff
@@ -565,109 +565,59 @@ long_max = cc.compute_int(
-        ['mallinfo2',         '''#include <malloc.h>'''],
-        ['execveat',          '''#include <unistd.h>'''],
-        ['close_range',       '''#include <unistd.h>'''],
-        ['epoll_pwait2',      '''#include <sys/epoll.h>'''],
-        ['mount_setattr',     '''#include <sys/mount.h>'''],
-        ['move_mount',        '''#include <sys/mount.h>'''],
-        ['open_tree',         '''#include <sys/mount.h>'''],
-        ['fsopen',            '''#include <sys/mount.h>'''],
-        ['fsconfig',          '''#include <sys/mount.h>'''],
-        ['fsmount',           '''#include <sys/mount.h>'''],
-        ['getdents64',        '''#include <dirent.h>'''],
-        ['pidfd_spawn',       '''#include <spawn.h>'''],
-        ['strerrorname_np',   '''#include <string.h>'''],
-        ['getrandom',         '''#include <sys/random.h>'''],
-        ['quotactl_fd',       '''#include <sys/quota.h>'''],
-        ['setxattrat',        '''#include <sys/xattr.h>'''],  # no known header declares setxattrat
-        ['removexattrat',     '''#include <sys/xattr.h>'''],  # no known header declares removexattrat
+        ['set_mempolicy',     '''#include <sys/syscall.h>'''],  # declared at numaif.h provided by libnuma, which we do not use
+        ['get_mempolicy',     '''#include <sys/syscall.h>'''],  # declared at numaif.h provided by libnuma, which we do not use
+        ['strerrorname_np',   '''#include <string.h>'''],       # since glibc-2.32
+        ['mallinfo2',         '''#include <malloc.h>'''],       # since glibc-2.33
+        ['execveat',          '''#include <unistd.h>'''],       # since glibc-2.34
+        ['close_range',       '''#include <unistd.h>'''],       # since glibc-2.34
+        ['epoll_pwait2',      '''#include <sys/epoll.h>'''],    # since glibc-2.35
+        ['fsconfig',          '''#include <sys/mount.h>'''],    # since glibc-2.36
+        ['fsopen',            '''#include <sys/mount.h>'''],    # since glibc-2.36
+        ['fsmount',           '''#include <sys/mount.h>'''],    # since glibc-2.36
+        ['mount_setattr',     '''#include <sys/mount.h>'''],    # since glibc-2.36
+        ['move_mount',        '''#include <sys/mount.h>'''],    # since glibc-2.36
+        ['open_tree',         '''#include <sys/mount.h>'''],    # since glibc-2.36
+        ['pidfd_open',        '''#include <sys/pidfd.h>'''],    # since glibc-2.36
+        ['pidfd_send_signal', '''#include <sys/pidfd.h>'''],    # since glibc-2.36
+        ['pidfd_spawn',       '''#include <spawn.h>'''],        # since glibc-2.39
+        ['sched_setattr',     '''#include <sched.h>'''],        # since glibc-2.41
+        ['ioprio_get',        '''#include <sched.h>'''],        # no known header declares ioprio_get
+        ['ioprio_set',        '''#include <sched.h>'''],        # no known header declares ioprio_set
+        ['rt_sigqueueinfo',   '''#include <signal.h>'''],       # no known header declares rt_sigqueueinfo
```

Comment:
hmm, this one is apparently unused after #36494, and hence can be removed directly?